### PR TITLE
Add installed version to status field of apicurio crd

### PR DESCRIPTION
**Pull request by JameelB**
## What
Add the version of the Apicurio installed within the `status` field of the Apicurio CRD.

## Why
Ensure that the version of the product installed by the Apicurio operator is reflected in the `status.version` field.

## Verification Steps
1. Run `make cluster/prepare`
2. Run the operator locally.
3. Create the Apicurio custom resource
4. Ensure that the correct version is shown within the `status` field of the Apicurio custom resource

## Checklist:

- [x] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member

### CODE REVIEW

**Comment by odra** [comment]
:eye:

**Review by odra — APPROVED** [review]

File: pkg/apis/integreatly/v1alpha1/apicuriodeployment_types.go [hunk 1]
```diff
@@ -13,18 +13,19 @@ const (
...
 
 type ApicurioDeploymentSpec struct {
-	Version         string     `json:"version"`
-	AppDomain       string     `json:"app_domain"`
-	Template        string     `json:"template"`
-	ExternalAuthUrl string     `json:"external_auth_url"`
-	AuthRealm       string     `json:"auth_realm"`
-	JvmHeap         [2]string  `json:"jvm_heap"`
-	MemLimit        [2]string  `json:"mem_limit"`
+	Version         string    `json:"version"`
```
Comment (maleck13) [inline pkg/apis/integreatly/v1alpha1/apicuriodeployment_types.go:16]: I think we should remove version from here. The operator should decide the version not the user

**Review by maleck13 — CHANGES_REQUESTED** [review]
see comment above

**Comment by JameelB** [comment]
@odra @maleck13 Closing this PR as this will be replaced by Apicurito in the new version of Fuse.

**Pull request closed without merging**